src/js/components/viewer/models/audio-media-object.jsx

```diff
@@ -123,67 +129,85 @@ var AudioMediaObject = React.createClass({
 
     },
 
-    play: function() {
+    audio5Play: function(mediaObject, streamUrl) {
 
-        var mediaObject = this.props.data.mediaObject._obj;
+        var self = this;
 
-        if(!this.state.playing) {
-            return;
+        var volume = mediaObject.volume;
+        if (isNaN(volume)) {
+            volume = 100;
         }
+        volume = volume / 100;
+
+        self.state.player = new Audio5({
+            throw_errors: false,
+            format_time: false,
+            ready: function() {
+                if(!self.state.playing) {
+                    return;
+                }
 
-        var self = this;
+                this.load(streamUrl);
 
-        soundCloud.streamUrl(mediaObject.url, function(err, streamUrl) {
-            if(err) {
-                throw err;
-                return;
-            }
+                // APEP TODO resolve audio volume bug then we can turn this back on
+                this.volume(volume);
 
-            var volume = mediaObject.volume;
-            if (isNaN(volume)) {
-                volume = 100;
-            }
-            volume = volume / 100;
-
-            self.state.player = new Audio5({
-                throw_errors: false,
-                format_time: false,
-                ready: function() {
-                    if(!self.state.playing) {
-                        return;
-                    }
+                self.unlockCuePointsForMediaObject(mediaObject);
 
-                    this.load(streamUrl);
+                this.play();
 
-                    this.volume(0);
+                //APEP tween volume TODO Turn back on with time to diagnose
```

Comment:
This is purposeful, we shall have to come back to the volume tween when we have a spare minute.